software/dhrystone/Makefile

```diff
@@ -8,7 +8,7 @@ DHRY_SRCS := dhry_1.c dhry_2.c
 DHRY_CFLAGS := -O2 -DTIME -fno-inline -fno-builtin-printf -Wno-implicit -mcmodel=medany -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI)
 
 XLEN ?= 32
-CFLAGS := -Os -fno-common  -mcmodel=medany -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI)
+CFLAGS := -O3 -fno-common  -mcmodel=medany -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI) -mexplicit-relocs -DNOENUM -falign-functions=4
```

Comment:
I think this -O3 is overridden by the -O2 in DHRY_CFLAGS above?

Reply:
Looks like the performance is the same with -O2 and -O3, but I fixed it anyway.